plugins/roll.py

```diff
@@ -25,9 +25,11 @@ def roll_number(message, number):
                 message.reply("{} :chuckles:".format(random))
         else:
             message.reply('Spa leur smh')
-    elif number > 1:
+    elif number > 2:
         random = randint(1, number)
         message.reply(random)
+    elif number == 1:
+        message.reply(":99:")
     else:
         message.reply(number + ' is not a valid number for roll command')
```

Comment:
```suggestion
        message.reply('{number} is not a valid number for roll command'.format(number=number))
```
Faudrait aussi corriger la unhandled exception